### DISCUSSION TITLE
Problem after image update (syno docker)

### DISCUSSION THREAD

**Discussion by ppeterrr** [qdm12/gluetun#919] [Q&A] [answered]
[edit] Aha :P i fixed it by removing the old entries. It works again 

Hi sorry to disturb you. I updated the gluetun container with wachtower on my synology.
Got a lot of error's, but in the log it was clear for me to change the variable names to those who are new.
But i'm stuck with one  error

2022/03/29 12:12:50 ERROR VPN settings: provider settings: server selection: the country specified is not valid: one or more values is set but there is no possible value available
-- | --

![image](https://user-images.githubusercontent.com/96293249/160590285-c7e5eafa-ebe9-436e-b720-0bb3ac810a5c.png)

Not sure what to do, or what is really changed. It has been runing oke for weeks :(

**Comment by qdm12** [comment, marked as answer]
> Aha :P i fixed it by removing the old entries. It works again

All good! Yes the code is decoupled to check every setting given, even if it's not meant to be used. I think this is useful to prevent unexpected surprises as well 😉 Happy tunneling!
